modules/service-directory/README.md

```diff
@@ -90,20 +90,59 @@ module "dns-sd" {
 }
 # tftest modules=2 resources=5 inventory=dns.yaml
 ```
-<!-- BEGIN TFDOC -->
 
+### Services with endpoints using private network access
```

Comment:
Private Network Access

Reply:
Official branding for _private network access_ is lowercase, unlike (for example) _Private Service Connect_.

Reply:
I would still make it uppercase to flag it's a product feature.

Reply:
Ok!